.github/workflows/platform_cd_workflow.yml

```diff
@@ -94,6 +94,7 @@ jobs:
         shell: bash
         run: |
             [ -d "src/${{ inputs.model_type }}/common/" ] && cp -r src/${{ inputs.model_type }}/common/* model/${{ inputs.model_type }}/online
+        continue-on-error: true
```

Comment:
do we know if the azdo `continue-on-error` still produces a warning/error log? if not, it may be useful to log it via the below bash command:

```
 if [ -d "src/${{ inputs.model_type }}/common/" ]; then
              cp -r src/${{ inputs.model_type }}/common/* model/${{ inputs.model_type }}/online
            else
              echo "Warning: Directory src/${{ inputs.model_type }}/common/ does not exist" >&2
            fi
  ```